src/config/exception.py

```diff
@@ -177,13 +196,14 @@ def raise_http_exception(e: Exception, msg: str = None, data: Any = None):
     403: ForbiddenException,
     404: NotFoundException,
     406: NotAcceptableException,  # No DuplicateUserException
+    422: UnprocessableClientException,
     429: TooManyRequestsException,
     500: ServerException,
 }
 
 
 def raise_http_exception_by_status_code(status_code: int, msg: str = None, data: Any = None):
     if status_code in status_code_mapping:
-        raise_http_exception(status_code_mapping[status_code](msg, data))
+        raise_http_exception(status_code_mapping[status_code](msg=msg, data=data))
```

Comment:
According to the params' order, the params value should be specified, otherwise it will become ```func(msg=msg, code=data)```, and it's not my purpose.
<img width="582" alt="Screenshot 2024-12-22 at 11 29 36 PM" src="https://github.com/user-attachments/assets/1e1a1fd1-bbf2-487b-b821-00ae4a5de512" />